week-3/Homework/mandatory/7-password-validator.js

```diff
@@ -21,15 +21,88 @@ Expected Result:
 PasswordValidationResult=  [false, false, false, false, true]
 
 */
-
+// let alph = /[a-z]/;
+// let specialChar = /[".", "!", "#", "%", "$"]/
 function validatePasswords(passwords) {
+  let newArr = passwords.map(Element =>
+    (Element.indexOf("!") >= 0 || 
```

Comment:
This is an interesting solution! It is correct, but how could we improve it?

Hint: you could try researching how to use a *regex*: here's some [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) to get you started.